SimonImanigirimpuhwe/my-brand-website
# #30 ft(darkmode): activate toggle switch to dark mode

#### What does this PR do?

- Activate dark mode toggle switch

#### Description of Task to be completed?

- Add default color behaviour

- Add scripts to activate the toggle switch


#### How should this be manually tested?

- Go to any page and click on the toggle switch button

- Then you should be able to see the background color and text color changed respectively  to activate dark mode reading

#### Any background context you want to provide?

- N/A

#### What are the relevant Trello stories?

- [#30](https://trello.com/c/mmbQxXqG/30-toggle-dark-mode)

#### Screenshots (if appropriate)



#### Questions:

- N/A
![toggle-switch-dark-mode](https://user-images.githubusercontent.com/54619678/94365321-1cb8c200-00d0-11eb-9e19-6fb68b4729b6.png)
